ParrotOS Appliance

brentstewart opened the pull request:
I've created an appliance that installs ParrotOS (a Linux security distribution).  Two files are added:  the gns3a file in the appliance diretory and the icon in the symbol directory.  I've tested locally and it's good.  One note: we're categorizing into router, switch, firewall, and guest.  I think we're getting enough appliances that it would be worthwhile to expand the categories.  I propose something like "traffic engineering" for things like compression  (Riverbed) and load balancing, plus a category for "security" (which would include ParrotOS, Kali and Security Onion.

Thanks for all you do!

================ 

Before submitting a pull request, please check the following.

---
When updating an **existing** appliance:
- [ ] The new version is on top.
- [ ] The filenames in the "images" section are unique, to avoid appliances / version overwriting each other.
- [ ] If you forked the repo, running check.py doesn't drop any errors for the updated file.
---
When creating a **new** appliance:
- It's tested locally, i.e.
  - [ ] You dragged an instance into a project on your box, got it installed (if necessary), and did some basic network checks (ping, UI reachable, etc.).
  - [ ] GNS3 VM can run it without any tweaks.
  - [ ] The device is in the right category: router, switch, guest (hosts), firewall
  - [ ] You filled in as much info as possible (checks the schemas and other appliance files for some guidance).
- [ ] When adding a container: it builds on Docker Hub and can be pulled.
- [ ] The filenames in the "images" section are unique (to avoid appliances and/or versions overwriting each other).
- [ ] If you forked the repo, running check.py doesn't drop any errors for the new file.
- [ ] *Optional: a symbol has been created for the new appliance.*

brentstewart commented:
Just curious - haven't seen this pulled - did I do something wrong?

grossmj commented:
Hi @brentstewart, thanks for your contribution however I only see the symbol files, not the .gns3a file.

brentstewart commented:
Well, that's embarrassing.  I've added the gns3 file.  Also, I added SecurityOnion a while back and I've updated the symbol.  Same image, but I've sized it more consistently with other symbols.
